src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5242,9 +5248,28 @@ namespace OutputReportTabular {
         // clear the ATU accumulation variables
         ATUHeat = 0.0;
         ATUCool = 0.0;
+        // clear the Zone Eq accumulatoin variables
+        ZoneEqHeatorCool = 0.0;
+        // clear the ATU Delta accumulation variables
+        ATUDeltaHeat = 0.0; 
+        ATUDeltaCool = 0.0;
         //--------------------
         //     ANNUAL
         //--------------------
+        // HVAC annual heating by ATU
+        // HVAC annual cooling by ATU
+        for (iunit = 1; iunit <= NumAirDistUnits; ++iunit) {
+            // HVAC equipment should already have the multipliers included, no "* mult" needed (assumes autosized or multiplied hard-sized air flow).
+            curZone = AirDistUnit(iunit).ZoneNum;
+            if ((curZone > 0) && (curZone <= NumOfZones)) {
+                ZonePreDefRep(curZone).SHGSAnHvacATUHt += AirDistUnit(iunit).HeatGain;
+                ATUDeltaHeat(curZone) += AirDistUnit(iunit).HeatGain;
+                ZonePreDefRep(curZone).SHGSAnHvacATUCl -= AirDistUnit(iunit).CoolGain;
+                ATUDeltaCool(curZone) -= AirDistUnit(iunit).CoolGain;
+                ATUHeat(curZone) = AirDistUnit(iunit).HeatRate;
+                ATUCool(curZone) = -AirDistUnit(iunit).CoolRate;
```

Comment:
These need to accumulate `+=` and `-=` because there can be more than one `AirDistUnit` in a zone. This is something I missed back when adding the ability to serve a zone with more than one airloop.

Reply:
This one has now been addressed.